library/src/main/java/com/mikepenz/materialdrawer/util/AbstractDrawerImageLoader.java

```diff
@@ -17,8 +17,8 @@ public void set(ImageView imageView, Uri uri, Drawable placeholder, String tag)
         set(imageView, uri, placeholder);
         //this won't do anything
         Log.i("MaterialDrawer", "you have not specified a ImageLoader implementation through the DrawerImageLoader.init(IDrawerImageLoader) method");
```

Comment:
I would prefer the single log message like 

"You have not specified a ImageLoader implementation through the DrawerImageLoader.init() method, or you are still overriding the deprecated method set(ImageView iv, Uri u, Drawable d) instead of set(ImageView iv, Uri u, Drawable d, String tag)" 

Could you please adjust this before I'll merge